File: src/model/chimpos/Carnotron.java
package model.chimpos;

import model.ataque.Ataque;
import model.ataque.CañonConico;
import model.ataque.RayoVeloz;

import java.util.ArrayList;
import java.util.Arrays;

public class Carnotron extends Chinpokomon {

    public Carnotron(String nombre, Integer nivel) {
        super(nombre, nivel, 20, new ArrayList<Ataque>(Arrays.asList(new RayoVeloz(), new CañonConico())));
    }

}

File: python/model/chipos/Carnotron.py
from model.ataque.CanionConico import CanionConico
from model.ataque.RayoVeloz import RayoVeloz
from model.chipos.Chipokomon import Chipokomon


class Carnotron(Chipokomon):
    def __init__(self, nombre, nivel, tipo):
        Chipokomon.__init__(self, nombre, nivel, 20,[RayoVeloz, CanionConico], tipo)

    def __str__(self):
        return 'Carnotron'



